src/ocaml-server/main.ml

```diff
@@ -1,12 +1,13 @@
 open Lwt.Infix
 open Cohttp_lwt_unix
+open Websocket
 
-
-type flag = { modified : bool ref}
-let file_flag = {modified = ref false}
+let modified = ref true
 let visualize = ref ""
 let no_file = ref false
 let port = ref 3030
+let path_to_ipm = (Sys.getenv "MI_IPM")
```

Comment:
You should probably use a `try` block here, so that the server doesn't crash if `MI_IPM` isn't set. We probably do want it to exit in this case, but then it should print a helpful message to the user and exit gracefully.

Reply:
Did wrap it in a try block! Thanks!